bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss

```diff
@@ -16,3 +16,5 @@ $color-heading: #4E525E !default;
 
 $color-link: #3E9DD6 !default;
 $color-link-hover: darken($color-link, 15%) !default;
+
+$color-wb-controls-background: #212121 !default;
```

Comment:
Should be moved to the component specific css (`.../components/whiteboard/slide-controls/styles.scss`).